operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -416,6 +416,9 @@ Examples:
   "querier" "querier"
   "query-frontend" "query_frontend"
   "query-scheduler" "query_scheduler"
+  "ruler-querier" "ruler_querier"
+  "ruler-query-frontend" "ruler_query_frontend"
+  "ruler-query-scheduler" "ruler_query_scheduler"
```

Comment:
Unless I'm very much mistaken, it looks like templates/ruler-query-scheduler  folder is not yet checked in?

Reply:
You are correct thx - I've pushed the latest update